socialhome/templates/account/signup.html

```diff
@@ -13,6 +13,7 @@
 
       <p>{% blocktrans %}Already have an account? Then please <a href="{{ login_url }}">sign in</a>.{% endblocktrans %}</p>
 
+      {% if is_open_for_signup %}
```

Comment:
No need to check this - if the signup is closed it will show "signups are closed" anyway.

![selection_342](https://cloud.githubusercontent.com/assets/1174866/23874864/e7d4febe-083f-11e7-917c-caf9dbc1020d.png)